[Wrangler] Add information about wrangler login to a remote machine (#5981)
cloudflare/cloudflare-docs: content/workers/wrangler/get-started.md

Open the browser, log in to your account, and select **Allow**. This will send an OAuth Token to Wrangler so it can deploy your scripts to Cloudflare.

> ### Using `wrangler login` in a remote machine

Reviewer: ```suggestion
### Using `wrangler login` on a remote machine
```

Reviewer 2: Aren't these grammatically equivalent in this instance?

Reviewer: You'll typically find `on a server/computer` to be the more common usage, `in a server/computer/remote machine` implies being physically inside it with the RAM/CPU

Author: https://github.com/cloudflare/cloudflare-docs/pull/5981/commits/76cacc2478f9d91726aa2579d6b5aef32fd9d6a8

> If you are using wrangler from a remote machine, but run the login flow from your local browser, after logging in you will see the following error message: `This site can't be reached`.

Reviewer 3: ```suggestion
If you are using Wrangler from a remote machine, but run the login flow from your local browser, you will receive the following error message after logging in:`This site can't be reached`.
```

> If you are using wrangler from a remote machine but run the login flow from your local browser, you will get an error after logging in that `This site can't be reached`. To finish the login flow, you can use `curl` or an equivalent request library on the remote machine to fetch this `localhost` url.

Reviewer: ```suggestion
If you are using wrangler from a remote machine, but run the login flow from your local browser, after logging in you will see the following error message: `This site can't be reached`. 

To finish the login flow, you can use `curl` or an equivalent request library on the remote machine to fetch this `localhost` url.
```

Reviewer 2: This makes sense and is more concise.

```sh
$ wrangler login
 ⛅️ wrangler 2.1.6
-------------------
Attempting to login via OAuth...
Opening a link in your default browser: https://dash.cloudflare.com/oauth2/auth?xyz...
```

> In a second terminal session:

Reviewer: ```suggestion
In a second terminal session, run:
```

Author: https://github.com/cloudflare/cloudflare-docs/pull/5981/commits/76cacc2478f9d91726aa2579d6b5aef32fd9d6a8

> ```

Reviewer: ```suggestion
```sh
```

Author: https://github.com/cloudflare/cloudflare-docs/pull/5981/commits/76cacc2478f9d91726aa2579d6b5aef32fd9d6a8

Open the browser, log in to your account, and select **Allow**. This will send an OAuth Token to Wrangler so it can deploy your scripts to Cloudflare.

### Use `wrangler login` on a remote machine

If you are using Wrangler from a remote machine, but run the login flow from your local browser, you will receive the following error message after logging in:`This site can't be reached`.

> To finish the login flow, you can use `curl` or an equivalent request library on the remote machine to fetch this `localhost` URL.

Reviewer 3: ```suggestion
To finish the login flow, run `wrangler login` and go through the login flow in the browser:
```

```sh
$ wrangler login
 ⛅️ wrangler 2.1.6
-------------------
Attempting to login via OAuth...
Opening a link in your default browser: https://dash.cloudflare.com/oauth2/auth?xyz...
```

> In a second terminal session, run:

Reviewer 3: ```suggestion
Leave the login flow active. Open a second terminal session. In that second terminal session, use `curl` or an equivalent request library on the remote machine to fetch this `localhost` URL:
```

> Leave the login flow active. Open a second terminal session. In that second terminal session, use `curl` or an equivalent request library on the remote machine to fetch this `localhost` URL:

Reviewer 3: ```suggestion
The browser login flow will redirect you to a `localhost` URL on your machine.

Leave the login flow active. Open a second terminal session. In that second terminal session, use `curl` or an equivalent request library on the remote machine to fetch this `localhost` URL. Copy and paste the `localhost` URL that was generated during the `wrangler login` flow and run:
```

```sh

> $ curl http://localhost:8976/oauth/callback?code=xyz...

Reviewer 3: ```suggestion
$ curl <LOCALHOST_URL>
```
